bcda/suppression/suppression.go

```diff
@@ -133,13 +135,21 @@ func (importer OptOutImporter) ImportSuppressionData(metadata *optout.OptOutFile
 			importer.Logger.Error(err)
 			return err
 		}
+		switch suppression.PrefIndicator {
+		case "Y":
+			optInCount++
+		case "N":
+			optOutCount++
+			return nil
+		}
 		return nil
 	})
 
 	if err != nil {
 		importer.updateImportStatus(metadata, optout.ImportFail)
 		return err
 	}
+	importer.Logger.WithFields(logrus.Fields{"created_opt_outs_count": optOutCount, "created_opt_ins_count": optInCount}).Info()
```

Comment:
Could we add a small message here like "Successfully imported file"? Just so we have something human-readable.

Reply:
absolutely!